packages/prettier-plugin-sql/package.json

```diff
@@ -13,9 +13,9 @@
   },
   "devDependencies": {
     "@prairielearn/tsconfig": "workspace:^",
-    "@types/node": "^18.17.4",
-    "prettier": "3.0.1",
-    "typescript": "^5.1.6"
+    "@types/node": "^18.17.12",
+    "prettier": "3.0.3",
```

Comment:
```suggestion
    "prettier": "^3.0.3",
```

Reply:
We always pin prettier:

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.
> [(source)](https://prettier.io/docs/en/install#summary)

This matters less with lockfiles, but still 🤷